### DISCUSSION TITLE
rendering engine

### DISCUSSION THREAD

**Discussion by iameagle201** [godotengine/godot-proposals#8924] [Engine Core]
If read it correctly, godot is it's own rendering engine. Blender has options to use different rendering engines like cycles and eevee, and possibly the moonray renderer. Is there any way to add other rendering engines to godot or plans to add an option like how blender does it?

**Comment by AThousandShips** [comment]
You can already select between [three options](https://docs.godotengine.org/en/stable/about/list_of_features.html#rendering)? You can write your own backend but it's quite difficult

**Comment by Calinou** [comment]
Blender's rendering engines and Moonray are designed for offline rendering, not real-time usage in games. They are way too slow for that use case to be realistic (even real-time pathtraced games use a different approach).

**Reply by abdulhakam** [reply]
> Blender's rendering engines and Moonray are designed for offline rendering, not real-time usage in games. They are way too slow for that use case to be realistic (even real-time pathtraced games use a different approach).

I have a question about blender, is it possible to somehow add EEVEE engine as a plugin renderer or something similar to godot as it is almost realtime and if we reduce the number of passes per frame, we can get very good framerate even on potatoes(although the ouput will then have something like a "low light grain" effect natively, but it can be almost ignored at high framerates).

**Reply by Calinou** [reply]
While EEVEE is based on rasterization, it's still oriented towards offline rendering. It can't really render interactive framerates in real world projects on most GPUs.
